daml-script/export/src/main/scala/com/daml/script/export/Encode.scala

```diff
@@ -105,7 +104,7 @@ private[export] object Encode {
   }
 
   private def encodePartyBinding(party: Party, binding: String): Doc =
-    s"let $binding = lookupParty" &: quotes(Doc.text(Party.unwrap(party))) :& "parties"
+    s"let $binding = lookupParty" &: quotes(Party.unwrap(party)) :& "parties"
```

Comment:
Unrelated to this PR (this is the first time I look at this code): Shouldn't we escape party names? Or are there strict rules on what constitutes a party name?

Reply:
from the [spec](../tree/main/daml-lf/spec/daml-lf-1.rst) : 
```
PartyId strings
     PartyIdString  ∈  [a-zA-Z0-9:\-_ ]{1,255}      -- PartyIdString
```